### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by pathaks1**
- Page to add clinicians redirects if person signed in is not a clinician
- All clinician dashboard pages only load if the person is a clinician
- Added a "child ID", which is a small number unique to each child so that clinicians can differentiate potential patients with the same first name; user context also reflects this now
- Fixed Microsoft sign in on clinician dashboard (needed to change some scopes and data access)
- Added sign out functionality on the mobile app
- Added reminder to only input first names and also a function that forcefully only takes the first "word" of a name (reflected in onboarding and account page)
- Fixed favicon and site title/description

### CODE REVIEW

File: clinician-dashboard/app/dashboard/[slug]/page.tsx [hunk 1]
```diff
@@ -11,10 +11,12 @@ const Dashboard = () => {
     const supabase = createClient();
 
     const [clinician, setClinician] = useState<any>();
+    const [groupId, setGroupId] = useState<any>();
     const [children, setChildren] = useState<any[]>([]);
     const [selectedChildID, setSelectedChildID] = useState<any>();
     const [childReports, setChildReports] = useState<any[]>([]);
     const [redirectBool, setRedirect] = useState(false);
+    const [checkedAcct, setChekcedAcct] = useState(false);
```
Comment (baylee-rl) [inline clinician-dashboard/app/dashboard/[slug]/page.tsx:19]: haha tiny typo in `setChekcedAcct` but technically non blocking

File: clinician-dashboard/app/page_components/LoginBox.tsx [hunk 2]
```diff
@@ -12,8 +12,8 @@ export default function LoginBox() {
             <OAuthButton provider="google"/>
             <hr className='m-5 bg-black dark:bg-white'/>
             <div className='text-center'>
-                <p className="text-black dark:text-white">Any questions? <a className="underline text-blue-500 cursor-pointer">Contact us.</a></p>
+                <p className="text-black dark:text-white">Any questions? <a className="underline text-blue-500 cursor-pointer" href="mailto:team.riceapps@gmail.com">Contact us.</a></p>
```
Comment (baylee-rl) [inline clinician-dashboard/app/page_components/LoginBox.tsx:15]: i had no idea this email existed lol, do we have the login to it? and if not, do ppl actually check this
alternatively we could set up a new email that forwards to you and i's emails maybe?

**Review by baylee-rl — APPROVED** [review]
some non-blocking comments, also noticing tons of places we need to implement better error handling oops 😭 but I think we can just do that later

File: clinician-dashboard/app/page_components/LoginBox.tsx (hunk 2 above)
Comment (pathaks1) [reply clinician-dashboard/app/page_components/LoginBox.tsx:15]: yea starting this summer board is checking it pretty regularly, and me and Ahmed also have forwarding setup. If you would like the email/pw I can forward it lol

File: clinician-dashboard/app/dashboard/[slug]/page.tsx (hunk 1 above)
Comment (pathaks1) [reply clinician-dashboard/app/dashboard/[slug]/page.tsx:19]: oops lmao

File: clinician-dashboard/app/page_components/LoginBox.tsx (hunk 2 above)
Comment (pathaks1) [reply clinician-dashboard/app/page_components/LoginBox.tsx:15]: otherwise I'm also fine with creating a new email just for us

**Pull request merged**